src/traffic_cop/traffic_cop_util.cpp

```diff
@@ -51,11 +53,27 @@ std::unique_ptr<planner::AbstractPlanNode> TrafficCopUtil::Optimize(
       auto types = order_by->GetOrderByTypes();
       auto exprs = order_by->GetOrderByExpressions();
       for (size_t idx = 0; idx < order_by->GetOrderByExpressionsSize(); idx++) {
+        if (exprs[idx].Get()->GetExpressionType() == terrier::parser::ExpressionType::VALUE_CONSTANT) {
+          common::ManagedPointer<parser::ConstantValueExpression> constant_value_expression =
+              exprs[idx].CastManagedPointerTo<parser::ConstantValueExpression>();
+          int64_t column_id = constant_value_expression->GetInteger().val_;
```

Comment:
What happens if the `ConstantValueExpression` is not an integer?

Reply:
This is a bug, I will fix it.